jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -647,39 +647,46 @@ static irep_idt get_if_cmp_operator(const irep_idt &stmt)
   throw "unhandled java comparison instruction";
 }
 
-static member_exprt
-to_member(const exprt &pointer, const exprt &fieldref, const namespacet &ns)
+/// Build a member exprt for accessing a specific field that may come from a
+/// base class.
+/// \param pointer: The expression to access the field on.
+/// \param field_instruction: A getfield/setfield instruction produced by the bytecode
+///   parser.
+/// \param ns: Global namespace
+/// \return A member expression accessing the field, through base class
+///   components if necessary.
+static member_exprt to_member(
+  const exprt &pointer,
+  const fieldref_exprt &field_instruction,
```

Comment:
This isn't the instruction, rather it's its argument, so `field_reference` or just `fieldref`?

Reply:
Addressed in previous PR.